src/Compatibility/Microsoft.DotNet.Compatibility/ValidatePackage.cs

```diff
@@ -23,6 +23,9 @@ public class ValidatePackage : TaskBase
         [Required]
         public string RoslynAssembliesPath { get; set; }
 
+        [Required]
```

Comment:
Seems rude to add this as required.  Especially since in the future we expect to support multiple Assemblies in a single package. https://github.com/dotnet/sdk/issues/17364

Reply:
I marked it as required since we control the targets that call into the task, and we know that AssemblyName property is an MSBuild property that should always be defined, so I didn't see any harm on making it required. Anyway, I'm fine with marking it optional and default back to use PackageId if it wasn't passed in.

Reply:
Please make it optional.  We may want to remove this completely in the future.